commonui/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,36 @@
+<resources>
+    <string name="error_dialog_title">Error</string>
+    <string name="error_dialog_connectivity_title">Problema de conectividad</string>
+    <string name="error_dialog_unexpected_title">Error inesperado</string>
+    <string name="ok">De acuerdo</string>
+    <string name="try_again_button_label">Inténtelo de nuevo</string>
+    <string name="not_available">No disponible</string>
+
+    <!-- title and font icon -->
+    <string name="bank_account">Cuenta bancaria</string>
+    <string name="bank_card">Tarjeta de débito</string>
+    <string name="wire_account">Cuenta de transferencia bancaria</string>
+    <string name="paper_check">Cheque impreso</string>
+    <string name="prepaid_card">Tarjeta prepagada</string>
+    <string name="paypal_account">PayPal</string>
+    <string name="venmo_account">Venmo</string>
+    <string name="cash_pickup_mg">MoneyGram</string>
+
+    <string name="bank_account_font_icon">\ue900</string>
+    <string name="bank_card_font_icon">\ue903</string>
+    <string name="wire_account_font_icon">\ue90a</string>
+    <string name="paper_check_font_icon">\ue902</string>
+    <string name="prepaid_card_font_icon">\ue901</string>
+    <string name="paypal_account_font_icon">\ue906</string>
+    <string name="venmo_account_font_icon">\ue909</string>
+    <string name="cash_pickup_mg_font_icon">\ue905</string>
```

Comment:
Nothing to translate here for icons, remove it from translated files

Reply:
Removed